src/components/stake/StakeValidatorFilter.vue

```diff
@@ -0,0 +1,116 @@
+<template>
+    <div class="validator-filter">
+        <div class="validator-filter-wrapper">
+            <button class="validator-switch" :class="state === FilterState.TRUST?'selected':''"
```

Comment:
There's a more elegant way to implement conditional style classes:
`:class="{ selected: state === FilterState.TRUST }"`